docs/redoc/master/openapi.json

```diff
@@ -13011,16 +13007,18 @@
             ]
           },
           "sample": {
-            "description": "How many points to select and search within.",
+            "description": "How many points to select and search within. Default is 10.",
```

Comment:
I decided to make `sample` and `limit` optional for the sake of consistency with other endpoints.